packages/tools/pluggable-widgets-tools/bin/mx-scripts.js

```diff
@@ -77,6 +77,8 @@ function getRealCommand(cmd, toolsRoot) {
         case "test:e2e:ts":
         case "test:e2e:js":
             return `wdio ${join(toolsRoot, "test-config/wdio.conf.js")}`;
+        case "test:e2e:dev":
+            return `wdio ${join(toolsRoot, "test-config/wdio.conf.js")}`;
```

Comment:
Maybe set `DEBUG` environment here iso in every `package.json`? You can do it with cross-env

Reply:
Done!